packages/fxa-settings/src/models/Account.ts

```diff
@@ -133,6 +133,62 @@ export interface ProfileInfo {
   emails: Email[];
 }
 
+// Account attributes that can be persisted
+const PERSISTENT = {
+  accountResetToken: undefined,
+  alertText: undefined,
+  displayName: undefined,
+  email: undefined,
+  grantedPermissions: undefined,
+  hadProfileImageSetBefore: undefined,
+  lastLogin: undefined,
+  // This property is set when a user has changed their primary email address and
```

Comment:
We might want a follow up to either remove this or use it better in the React Cleanup epic - we have our own retry logic [like this](https://github.com/mozilla/fxa/blob/0a1cc35cc672ab1e9930c8a7300046599d0c72ec/packages/fxa-settings/src/pages/Signin/container.tsx#L659-L660), for example. Then we can see if there's other things we can remove, probably also a good place for looking at `isDefault`.

Reply:
I'll delete.  Looks like we don't rely on the account model for it like you pointed out.